pkg/database/database_util.go

```diff
@@ -138,6 +138,7 @@ func NewTestDatabaseWithCacher(tb testing.TB, cacher cache.Cacher) (*Database, *
 	}
 
 	// Re-enable logging.
+	db.db = db.db.LogMode(true)
```

Comment:
This enables _debug_ logging. Do we want that (over regular logging)?

Reply:
I needed this to debug this test, but will revert in general.